scss/_custom-forms.scss

```diff
@@ -58,7 +63,7 @@
   color: #eee;
   text-align: center;
   user-select: none;
-  background-color: #eee;
+  background-color: #FFFFFF;
```

Comment:
Color `#FFFFFF` should be written as `#FFF`<br>Color `#FFFFFF` should be written as `#ffffff`